### PR TITLE
#40 Fix single trainee detail page

### PR DESCRIPTION
**Pull request by Ismaelmurekezi**
### PR Description
This PR is for correcting issue related to single trainee detail page by adding missing  functionality. Also following best practices to avoid some error which was rising during test on github , improving user experience 

### Desciption of task on what was needed to be done
- Refactoring
- Refining User Interface
- Fixing some issues 

### How to test this PR
**Locally**
 -First ,you have to switch to fix-single-detail-page branch
 -Login as admin then under trainee-applicants page you choose view  
  Email:admin@example.com
  Password:password123
 
 -My branch: fix-single-trainee-detail-page
 
**Deploy** 
-For deployed one ,you click issue link, then under vercel  bot
-Click Preview , Login after creating account , then the rest is same as those of local 

https://atlp-devpulse-fn-git-fix-sin-0e7ca9-devpulses-projects-62c56cf1.vercel.app/

### Track PR
issue link: 
https://github.com/atlp-rwanda/atlp-devpulse-fn/pull/125

Before
Before

After

After

After3

### CODE REVIEW

**Review by Prince-Kid — APPROVED** [review]

File: .codeclimate.yml [hunk 1]
```diff
@@ -33,18 +33,18 @@ checks:
     config:
       threshold: 4
   similar-code:
-    enabled: true
+    enabled: false
```
Comment (mugishaj092) [inline .codeclimate.yml]: I think we need this restriction because it's help us to write better code

**Review by mugishaj092 — CHANGES_REQUESTED** [review]
Also rember to add other action button (Approve and Reject)

**Review by Ishimwe7 — COMMENTED** [review]
Please consider feedbacks given and resolve conflicts

File: .codeclimate.yml (hunk 1 above)
Comment (Ismaelmurekezi) [reply .codeclimate.yml]: Thanks,I have  solved that issues related to codeclimate

**Review by mugishaj092 — CHANGES_REQUESTED** [review]
add build in .env and remember to delete it after adding it in .env

**Review by mugishaj092 — APPROVED** [review]

**Comment by ncutixavier** [comment]
@Ismaelmurekezi  could you add credential in the PR description to test your feature

**Review by tysepa — APPROVED** [review]

**Pull request merged**